src/test/resources/com/puppycrawl/tools/checkstyle/grammar/javadoc/javadocTags/expectedTextBeforeJavadocTagsAst.txt

```diff
@@ -16,22 +16,22 @@ JAVADOC -> JAVADOC [0:0]
 |   |--WS ->   [3:15]
 |   `--DESCRIPTION -> DESCRIPTION [3:16]
 |       |--TEXT -> Int [3:16]
-|       `--NEWLINE -> \r\n [3:19]
+|       `--NEWLINE -> \n [3:19]
 |--LEADING_ASTERISK ->  * [4:0]
 |--WS ->   [4:2]
 |--JAVADOC_TAG -> JAVADOC_TAG [4:3]
 |   |--AUTHOR_LITERAL -> @author [4:3]
 |   |--WS ->   [4:10]
 |   `--DESCRIPTION -> DESCRIPTION [4:11]
 |       |--TEXT -> Walter White [4:11]
-|       `--NEWLINE -> \r\n [4:23]
+|       `--NEWLINE -> \n [4:23]
```

Comment:
Please explain why we lost `\r`.
It is good to have inputs in windows lineends

Reply:
We technically never had the `\r` to begin with.
Repo is linux but CI can be run on windows and windows converts to `\r\n`. To be able to run the same tests on both CIs, we had to hack away window's `\r\n`. We did this to both expected and actual results.
See: 
https://github.com/checkstyle/checkstyle/blob/master/src/test/java/com/puppycrawl/tools/checkstyle/AbstractTreeTestSupport.java#L55-L56
and
https://github.com/checkstyle/checkstyle/blob/c7ffe16d7e090cb5832aac65b874cf869de38d49/src/test/java/com/puppycrawl/tools/checkstyle/AbstractPathTestSupport.java#L72-L74
This file is not in `.gitattributes` so it was never being forced to `\r\n` on linux, hence me saying "We technically had the `\r` to begin with".

When I was copying over actual results to expected to pass tests, this change came in.